conans/client/conan_api.py

```diff
@@ -656,10 +656,9 @@ def info(self, reference_or_path, remote_name=None, settings=None, options=None,
         recorder = ActionRecorder()
         remotes = self.app.cache.registry.load_remotes()
         remotes.select(remote_name)
-        # FIXME: Using update as check_update?
-        self.app.python_requires.enable_remotes(check_updates=update, remotes=remotes)
+        self.app.python_requires.enable_remotes(update=True, remotes=remotes)
         deps_graph, conanfile = self.app.graph_manager.load_graph(reference, None, graph_info, build,
-                                                               update, False, remotes,
+                                                               update, update, remotes,
```

Comment:
The problem that I have with this change is that it modifies existing, documented behavior:

From the docs and the command line:

> -u, --update          Check updates exist from upstream remotes

It checks if they EXIST, but nowhere it says it will install and update them.

The ``info`` command is informational, it will produce the least changes it can to the conan cache. Fetching non-existing packages is necessary, to be able to parse them, but otherwise, it probably shouldn't be used to update the current installation, that will not be "Information" anymore.

I wouldn't have much problem in changing it, I agree the semantics might not be very clear, but I know that there were users relying and using this functionality.